packages/web/src/common/store/social/tracks/recordListen.ts

```diff
@@ -24,7 +24,7 @@ function* recordListen(action: { trackId: number }) {
   const track = yield* select(getTrack, { id: trackId })
   if (!userId || !track) return
 
-  if (userId === track.owner_id && (track.listenCount ?? 0) > 10) {
+  if (userId === track.owner_id && (track.play_count ?? 0) > 10) {
```

Comment:
what's the difference here?

Reply:
nice catch didnt mean to change this file actually. wonder if this is actually broken though, i dont think listenCount is a thing.